app/assets/stylesheets/govuk_publishing_components/components/helpers/_buttons.scss

```diff
@@ -0,0 +1,119 @@
+// global button styles
```

Comment:
I think most of the file if not all of it should be removed, as it only existed in static to style buttons that are not the component.